### ISSUE TITLE
Crashing when selecting Add-ons

### ISSUE THREAD

**Issue by treble20** [koying/SPMC#669]
I have installed the 16.5.5 64-bit version over the ARM32 version previously installed on my Shield and now every time I try to open my video add-on list, SPMC crashes back to the Shield homescreen.

The action when selecting my video add-ons is ActivateWindow(Videos,Addons,return). If I change this to Program, Music or Picture instead they all open, as does selecting an individual addon.

I have force closed SPMC and rebooted the Shield just in case, but no luck.

Any ideas please?

**Comment by koying** [comment]
> Any ideas please?

Sure. Upload and give me the id of the crash log

Now:


> SPMC crashes back to the Shield homescreen

and


> I have force closed SPMC

seem contradictory

**Comment by treble20** [comment]
Thanks. The ID is p76xf9kmm

What I meant by that is I started SPMC again and then force closed just to see if it made any difference when I started it again.

**Comment by koying** [comment]
Did that one crash?
If so, you didn't you receive the dialog asking to upload a crash log?

**Comment by treble20** [comment]
Yes, it crashed. Sometimes, not always, I get the 'Unfortunately SPMC has stopped' message but no dialog asking to upload a crash log. 

When my Addons is selected from the home screen, it's almost as if the Shield's home button has been pressed as it immediately returns to the Shield's homescreen
